src/katsdpcontroller/generator.py

```diff
@@ -1010,6 +1010,8 @@ def get_baseline_index(a1, a2):
                     "Number of visibilities that saturated",
                     name_regex=re.compile(rf"{escaped_name}\.[0-9]+\.xeng-clip-cnt"),
                     n_children=stream.n_substreams,
+                    auto_strategy=SensorSampler.Strategy.EVENT_RATE,
+                    auto_strategy_parameters=(GPUCBF_MIN_SENSOR_UPDATE_PERIOD, math.inf),
```

Comment:
To reduce duplicated code, I think it'll work to have a module-level constant

```python
FAST_SENSOR_UPDATE_PARAMS = {
    "auto_strategy": SensorSampler.Strategy.EVENT_RATE,
    "auto_strategy_parameters": (GPUCBF_MIN_SENSOR_UPDATE_PERIOD, math.inf),
}
```
and then here you could do
```suggestion
                    **FAST_SENSOR_UPDATE_PARAMS,
```

You'll just have to see what mypy thinks of it though. I think mypy complaints might be why I didn't use this pattern in katgpucbf, although it might also be a frog-in-boiling-water effect (initially there were only a couple of usages so it wasn't worth the effect, then applying it to one new sensor was always easier than doing the refactoring).